lib/Immediate/ImmediateImpl.h

```diff
@@ -34,7 +34,9 @@ namespace swift {
 
 namespace immediate {
 
-bool loadSwiftRuntime(StringRef runtimeLibPath);
+// Returns a handle to the runtime suitable for other 'dlsym' or 'dlclose' 
```

Comment:
Nitpick: please make this a proper doc comment, so that it shows up in Quick Help.